libs/model/commands/issue_asset.hpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
+ * http://soramitsu.co.jp
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef IROHA_ISSUE_ASSET_HPP
+#define IROHA_ISSUE_ASSET_HPP
+
+#include <model/model.hpp>
+#include <string>
+
+namespace iroha {
+  namespace model {
+
+    /**
+     * Add amount of asset to an account
+     */
+    struct IssueAsset : public Command {
```

Comment:
Just Issue is not really clear. In this case I need to understand from the naming what to do (issue), with what (asset). In case if we write just Issue - it is not clear from the name.